src/runtimes/rego/runtime.rs

```diff
@@ -21,17 +18,39 @@ impl<'a> Runtime<'a> {
         &mut self,
         settings: &PolicySettings,
         request: &ValidateRequest,
+        ctx_data: &context_aware::KubernetesContext,
     ) -> AdmissionResponse {
         let uid = request.uid();
 
         // OPA and Gatekeeper expect arguments in different ways. Provide the ones that each expect.
         let (document_to_evaluate, data) = match self.0.policy_execution_mode {
-            RegoPolicyExecutionMode::Opa => (
-                json!({
+            RegoPolicyExecutionMode::Opa => {
+                // Policies for OPA expect the whole `AdmissionReview`
```

Comment:
we can remove this comment as it is not relevant anymore.
the OPA policy expect just a "request" key containing the request object.

Reply:
removed